modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -61,26 +61,33 @@ export default class NestedAccordion extends Base {
 		} );
 	}
 
-	linkContainer() {
-		const containers = this.findElement( this.getSettings( 'selectors.accordionContentContainers' ) );
-		const accordionItems = this.findElement( this.getSettings( 'selectors.accordionItems' ) );
-		const lastContentContainer = containers[ containers.length - 1 ];
-		const lastAccordionItem = accordionItems[ accordionItems.length - 1 ];
+	linkContainer( event ) {
+		const { container } = event.detail,
+			view = container.view.$el,
+			id = container.model.get( 'id' ),
+			currentId = this.$element.data( 'id' );
 
-		lastAccordionItem.appendChild( lastContentContainer );
+		if ( id === currentId ) {
+			const containers = view.find( this.getSettings( 'selectors.accordionContentContainers' ) );
+			const accordionItems = view.find( this.getSettings( 'selectors.accordionItems' ) );
+			const lastContentContainer = containers[ containers.length - 1 ];
+			const lastAccordionItem = accordionItems[ accordionItems.length - 1 ];
 
-		this.updateListeners();
+			lastAccordionItem.appendChild( lastContentContainer );
+
+			this.updateListeners( view );
+		}
 	}
 
-	updateListeners() {
-		this.elements.$accordionTitles = this.findElement( this.getSettings( 'selectors.accordionItemTitles' ) );
-		this.elements.$accordionItems = this.findElement( this.getSettings( 'selectors.accordionItems' ) );
+	updateListeners( view ) {
+		this.elements.$accordionTitles = view.find( this.getSettings( 'selectors.accordionItemTitles' ) );
+		this.elements.$accordionItems = view.find( this.getSettings( 'selectors.accordionItems' ) );
 		this.elements.$accordionTitles.on( 'click', this.clickListener.bind( this ) );
 	}
 
 	bindEvents() {
 		this.elements.$accordionTitles.on( 'click', this.clickListener.bind( this ) );
-		elementorFrontend.elements.$window.on( 'elementor/nested-container/created', this.linkContainer.bind( this ) );
+		window.addEventListener( 'elementor/nested-container/created', this.linkContainer.bind( this ) );
```

Comment:
I changed this to`window.addEventListener` because I was trying different things, but now that we know what went wrong maybe we can revert to the original listener.